src/components/ui/textarea.stories.tsx

```
@@ -0,0 +1,71 @@
+
+export default meta;
+type Story = StoryObj<typeof Textarea>;
+
+export const Default: Story = {
+  args: {
+    placeholder: "Write your message here...",
+    rows: 4,
+  },
+};
+
+export const Error: Story = {
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)